indexer/indexer.gen.go

```diff
@@ -1,6 +1,6 @@
-// sequence-indexer v0.4.0 f06035d0f37c64eb9b9ea995af80088566898bb1
+// sequence-indexer v0.4.0 2f47ac5c56c1803ced8469e360dcf708670a94d5
 // --
-// Code generated by webrpc-gen@v0.25.3 with golang generator. DO NOT EDIT.
+// Code generated by webrpc-gen@v0.21.1 with golang generator. DO NOT EDIT.
```

Comment:
This is a downgrade, since we are now generating the client from `release` branch..